main.py

```diff
@@ -0,0 +1,172 @@
+                            Column("id", Integer, primary_key=True),
+                            Column("name", String(255)),
+                            Column("surname", String(255)),
+                            Column("age", Integer),
+                            Column("email", String(255)),
+                            Column("gender", String(255)),
+                            Column("language", String(255)),
+                            Column("country", String(255)),
+                            Column("occupation", String(255)),
+                            Column("phone", String(255)),
+                            Column("username", String(255)),
+                            Column("added_at", TIMESTAMP),
+                            Column("added_by", String(255))
+                            )
+
+        development_table = Table("development", meta,
+                                  Column("dev_id", Integer, autoincrement=True, primary_key=True),
+                                  Column("user_id", Integer, ForeignKey("users.id")),
+                                  Column("language", String(255)),
+                                  Column("language_2", String(255)),
+                                  Column("software_license", String(255)),
+                                  Column("os", String(255)),
+                                  Column("experience", Integer),
+                                  Column("added_at", TIMESTAMP),
+                                  Column("added_by", String(255), default='mimesis.Person'),
+                                  Column("income_usd", Integer)
+                                  )
+
+        companies_table = Table("companies", meta,
+                                Column("company_id", Integer, autoincrement=True, primary_key=True),
+                                Column("company", String(255), unique=True),
+                                Column("ceo", String(255)),
+                                Column("country", String(255)),
+                                Column("added_at", TIMESTAMP),
+                                Column("added_by", String(255))
+                                )
+        try:
+            engine = create_engine(
+                path.format(username, password, hostname, dbname),
+                echo=True)
+            meta.create_all(engine)
+            connection = engine.connect()
+            print('MySql connection is opened')
+        finally:
+            connection.close()
+            print('Tables users, development, companies were created. Connection is closed')
+
+    def df_creation(self, n):
+        self.users = self.users.append(self.__new_users(n))
+        self.devs = self.devs.append(self.__new_devs(n))
+        self.companies = self.companies.append(self.__new_companies())
+        while self.devs.shape[0] != n:
+            dif = n - self.devs.shape[0]
+            self.devs = self.devs.append(self.__new_devs(dif))
```

Comment:
please, move that code inside new devs generation method. There are 2 reasons to do that:
1) All the functions / methods should perform their tasks only;
2) This is causing a failure on multiple attempts to create a data frame